packages/ffe-form-react/src/InputGroup.js

```diff
@@ -64,11 +65,15 @@ class InputGroup extends Component {
                 {React.isValidElement(label) &&
                     React.cloneElement(label, { htmlFor: this.id })}
 
-                {typeof tooltip === 'string' && (
-                    <Tooltip onClick={onTooltipToggle}>{tooltip}</Tooltip>
-                )}
-                {tooltip === true && <Tooltip onClick={onTooltipToggle} />}
-                {React.isValidElement(tooltip) && tooltip}
+                {typeof tooltip === 'string' &&
+                    !helpText && (
+                        <Tooltip onClick={onTooltipToggle}>{tooltip}</Tooltip>
+                    )}
```

Comment:
Could we show a warning in the console or throw an error here if the user has provided both the `tooltip` and `helpText` props?  This way, it's easier to spot any potential errors.

Reply:
To what purpose? Why would anyone ever try to do that? The only reason the code had to care was the even-listener thing.

Reply:
I mean, I could add it, if you really think it's important, but to me it just feels like bloat.

Reply:
Well, to be fair that did make a couple of the checks further down redundant :)

Reply:
It's one of those things that often gets lost in translation, or one prop is left behind when you refactor or whatever. One of the great things I love about React as a library is all their helpful warnings during development. If you wrap the warning code in a dead-code-eliminatable check, there won't be bloat either.

Reply:
Also, some people might try to add both a tooltip AND a help text because of reasons - showing a warning would discourage them from doing so.

Reply:
I've added it and implementet it as the already existing warnings, see later comments.